jujugui/static/gui/src/app/views/environment.js

```diff
@@ -211,6 +211,16 @@ YUI.add('juju-view-environment', function(Y) {
           helpText.show();
           helpText.removeClass('shrink');
           includedPlus.classed('show', false);
+          helpText.one('.plus-service').on('click', function() {
```

Comment:
This is going to attach this click event every time there is a service update in the db so this needs to move elsewhere.